Lucid/Stores/LRUStore.swift

```diff
@@ -37,6 +37,8 @@ public final class LRUStore<E>: StoringConvertible where E: LocalEntity {
 
     private lazy var identifiersDispatchQueue = DispatchQueue(label: "\(ObjectIdentifier(self))", attributes: .concurrent)
 
+    private lazy var identifiersAsyncQueue = AsyncTaskQueue()
```

Comment:
I'm not sure why this was previously marked as `lazy`. I'm thinking we can make this a normal `let`.